docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should cite properly Gammapy.
+It is asked to cite the DOI of the latest Gammapy paper.
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This scheme and these two references are precise on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors, or `contributors`, that requires to be updated with time according to a
+general policy. This section defines what is a contributor to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (CI, GitHub configurations, gammapy-docs, release processes, etc),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra, gammapy-cat, gammapy-recipes),
+- contributing to the maintenance of our repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
+- organizing and leading hands-on sessions or schools,
+- managing our social media accounts,
+- and, de-facto, the members of the Coordination Committee.
+
+Any contribution on these topics renders the user as an official Gammapy contributor.
+
+.. important::
+    We strongly recommend that the contributors use always the same GitHub account, to have a unique
+    `ORCID <https://orcid.org/>`_ identifier and store it into your GitHub user profile, and inform the developers
+    of any change of affiliation.
+
+As Gammapy is an open project, we encourage users to provide feedback on the usage of Gammapy, on features, on issues
+via our communication channels.
+
+Definition of the `Maintainer`
+------------------------------
+
+In the Open Source community, a maintainer is generally mentioned in the metadata associated with a release and has to be mentioned.
+Usually, a software maintainer or package maintainer is one or more people who build source code into a binary package
+for distribution, commit patches, or organize code in a source repository.
+
+For Gammapy, by default, the maintainers are the Lead Developers. If in the future a task is dedicated to the creation
+of a release, the maintainer will be the person in charge of this task.
```

Comment:
Note that this is associated to the PyPi and GitHub access rights as well.

Reply:
> GitHub does not provide metadata.

That is not entirely true, e.g. for some definition of maintainer, there is code-owners:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners